x-pack/plugins/cases/public/components/configure_cases/connectors_dropdown.tsx

```diff
@@ -31,28 +38,23 @@ const EuiIconExtended = styled(EuiIcon)`
   margin-bottom: 0 !important;
 `;
 
+const FlexItemIconContainer = styled(EuiFlexItem)`
+  svg {
```

Comment:
hacky CSS to make the icons look as we want.

There's no reasonable way of doing this other way because of a weird rendering problem with triggersActionsUi and EuiIcon.